Northstar.Client/mod/resource/ui/menus/mod_settings.menu

```diff
@@ -0,0 +1,570 @@
+resource/ui/menus/mods_browse.menu
```

Comment:
Barichello started formatting res files so if you could pretty please do so too :), check [this for reference](https://github.com/R2Northstar/NorthstarMods/blob/main/Northstar.Client/mod/resource/ui/menus/map_select.menu)

Reply:
is there anything that does this automatically?